crates/cheatnet/tests/cheatcodes/declare.rs

```diff
@@ -1,95 +1,86 @@
+use std::collections::HashMap;
+
 use crate::common::state::create_cheatnet_state;
 use crate::common::{get_contracts, state::create_cached_state};
+use cheatnet::cheatcodes::declare::get_class_hash;
 use cheatnet::cheatcodes::{CheatcodeError, EnhancedHintError};
 use conversions::StarknetConversions;
+use scarb_artifacts::StarknetContractArtifacts;
 use starknet_api::core::ClassHash;
-use starknet_api::hash::StarkFelt;
-use starknet_api::stark_felt;
-
-static HELLO_STARKNET_CLASS_HASH: &str =
-    "0x05ecd2c5f5ca68a4dc1b945582c69434adfc8bedbd188f0146a35875a4791936";
-static CONSTRUCTOR_SIMPLE_CLASS_HASH: &str =
-    "0x05f5abdb98a7ec93a20ac4b2ed284979613bc71f2dec3df10e589ebe40c562b4";
+use test_case::test_case;
+
+fn class_hash_from_str(
+    contract_name: &str,
+    contracts: &HashMap<String, StarknetContractArtifacts>,
+) -> ClassHash {
+    let sierra = contracts.get(contract_name).unwrap();
+    get_class_hash(sierra.sierra.as_str()).unwrap()
+}
 
-#[test]
-fn declare_simple() {
+#[test_case("HelloStarknet")]
```

Comment:
Don't see a purpose in parametrizign that

Reply:
It's the main context of the test; another contract can be easily tested without touching the test itself
Can revert it though